docs/design/simple-rolling-update.md

```diff
@@ -0,0 +1,37 @@
+## Simple rolling update
+This is a lightweight design document for simple rolling update in ```kubectl```
+
+### Lightweight rollout
+Assume that we have a current replication controller named ```foo``` and it is running image ```image:v1```
```

Comment:
I'm ok if the multi-container case requires a config file.  I think this will hit the 80% use case pretty squarely.